pytorch_lightning/trainer/trainer.py

```diff
@@ -430,7 +430,7 @@ def __init__(
         log.detail(f"{self.__class__.__name__}: Initializing trainer with parameters: {locals()}")
         self.state = TrainerState()
 
-        gpu_ids, tpu_cores = self._parse_devices(gpus, auto_select_gpus, tpu_cores)
+        _, tpu_cores = self._parse_devices(gpus, auto_select_gpus, tpu_cores)
```

Comment:
Even the `_parse_devices` method won't be required :) 

Removing it in #12608

Reply:
Yes!